azure-iot-hub/protocol-generator.md

```diff
@@ -0,0 +1,24 @@
+# DPS Service Client
+> see https://aka.ms/autorest
+
+## Getting Started
+To build the DPS Service Client, simply install AutoRest in Node.js via `npm` (`npm install -g autorest`) and then run:
+> `autorest protocol-generator.md --add-credentials`
```

Comment:
>--add-credentials [](start = 34, length = 17)

nit: this can go in the python section of the yaml configuration so that you don't have CLI options and file options at the same time